pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -241,3 +258,88 @@ def cmd(self):
 
         cmd = (" ").join(cmd)
         return cmd
+
+
+class A11yCrawler(BokChoyTestSuite):
+    """
+    Sets up test environment with mega-course loaded, and runs pa11ycralwer
+    against it.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(A11yCrawler, self).__init__(*args, **kwargs)
+
+        self.pa11y_report_dir = os.path.join(self.report_dir, 'pa11ycrawler_reports')
+        self.imports_dir = path('test_root/courses/')
+        self.tar_gz_file = "https://github.com/edx/demo-test-course/archive/master.tar.gz"
+
+    def __enter__(self):
+        self.get_test_course()
+        super(A11yCrawler, self).__enter__()
+
+    def get_test_course(self):
+        """
+        Fetches the test course.
+        """
+        self.imports_dir.makedirs_p()
+        zipped_course = self.imports_dir + 'demo_course.tar.gz'
+
+        msg = colorize('green', "Fetching the test course from github...")
+        print msg
+
+        sh(
+            'wget {tar_gz_file} -O {zipped_course}'.format(
+                tar_gz_file=self.tar_gz_file,
+                zipped_course=zipped_course,
+            )
+        )
```

Comment:
This is going to fetch the test course every time your run the crawler, even if it hasn't changed. It also adds a dependency on an external resource at test runtime. Perhaps this could be pulled in by paver instead, and only executed if the file doesn't exist, or if it doesn't match an md5 sum?